docs/_data/components.yml

```diff
@@ -1,97 +1,100 @@
 ## Data for componnets list display page.
-  items:
-    - title: Action Bar
-      meta: action-bar
+items:
```

Comment:
those formatting changes should be done in a separate PR

Reply:
that means that prettierrc file is not used